### DISCUSSION TITLE
Welcome to laravel-inertia-react-shadcn-boilerplate Discussions!

### DISCUSSION THREAD

**Discussion by ariabid** [CodeRomeos/laravel-inertia-react-shadcn-boilerplate#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JY-TechIN** [comment]
When the new sidebar [shadcn sidebar](https://ui.shadcn.com/blocks) will be coming in this?
